asteriskserver/src/etc-asterisk/extensions.conf

```diff
@@ -550,6 +550,7 @@ exten => 2,1,Goto(trimet-transit-tracker,s,1)
 exten => 3,1,Goto(lib-account-line,s,1)
 exten => 4,1,Goto(lets-consider,s,1)
 exten => 5,1,Goto(random-number,s,1)
+exten => 6,1,Goto(jack-shit,s,1)
```

Comment:
comment this out since this is going to trunk
I know it's a pain to uncomment it each time, but that's the drawback of not using feature branches